ambry-store/src/main/java/com.github.ambry.store/BlobStoreCompactor.java

```diff
@@ -254,8 +254,8 @@ A single compaction job could be performed across multiple compaction cycles (if
   int getSwapSegmentsInUse() throws StoreException {
     String[] tempSegments = dataDir.list(TEMP_LOG_SEGMENTS_FILTER);
     if (tempSegments == null) {
-      throw new StoreException("Error occured while listing files in data dir:" + dataDir.getAbsolutePath(),
-          StoreErrorCodes.IOError);
+      throw new StoreException("Error occurred while listing files in data dir:" + dataDir.getAbsolutePath(),
```

Comment:
this probably still qualifies as an `IOError`

Reply:
Was this addressed?